redi-connect-front/public/locales/en/translation.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
I'm concerned this locale file is built with one single page in mind, and so will not scale up well when we extend it to cover all pages. Locale files are a major PITA to refactor later on (AFAIK IDEs don't have a neat function for this), and from experience it's worthwhile to spend some time right from the start getting it at least mostly right.

Some things to consider:
- Use different namespaces, and one locale file per namespace. Namespaces here could be "loggedInArea" and "loggedOutArea".
- Favour somewhat longer and more explicit names over shorter and ambiguous names.
- Use the [clean code tips about variable naming](https://github.com/ryanmcdermott/clean-code-javascript#variables)
- Think in terms of the existing component hierarchy we already have: logged in/out => pages => templates => organisms => molecules => atoms
- Where strings are frequently used across the entire app, put them in a "common" area

To get more hands on, here's an example of what I have in mind:

```
{
  footer: {

  }
  common: {
    categories: ,
    login: ,
    logout: ,
  }
  loggedInArea {
    header: {}
    dashboardPage: {},
    mentorProfilePage: {}
    menteeProfilePage: {}
    applicationsPage: {}
  }
  loggedOutArea {
    header: {}
    login: {}
    signup: {}
    landingPage: {
      hero: {

      }
      program: {
        steps: {}
      }
    }
  }
}
```

Reply:
done. I also added a `loggedInArea` and `common` key-value pair already to have the structure of the `translation.json` file + as a reminder. If you prefer not to have it atm I will delete it